docs/src/app/pages/DockedComposer/Email/ExampleEmail.js

```diff
@@ -24,6 +24,11 @@ const footerButtons = [
     icon: 'delete',
     key: 'delete_icon',
   },
+  {
+    customButton: <Button icon icon-container key="CustomButton">
```

Comment:
i meant setting it in the loop automatically for each customButton, but having to set it is fine since it is documented.